composer.json

```diff
@@ -15,7 +15,7 @@
         "email": "support@flownative.com"
     },
     "require": {
-        "neos/flow": "^4.0 || ^5.0 || ^6.0 || dev-master",
+        "neos/flow": "^4.0 || ^5.0 || ^6.0 || ^7.0 || dev-master",
```

Comment:
I did  not check (sorry!) but I am pretty sure this is no longer compatible with Flow 4, 5 & 6 with the changed migration signatures, is it?

Also, the change in `Classes/OAuthClient.php` – in which Flow versions is that working?